CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 3.0.6 – 2022-02-16
+### Fixed
+- Fix invalid X-APPLE-STRUCTURED-LOCATION on location update
+- Fix trashbin being unavaliable
+- FIx previously ignored DESCRIPTION;ALTREP property
```

Comment:
```suggestion
- Fix previously ignored DESCRIPTION;ALTREP property
```

Reply:
argh I will have to fix this in the next release :/